Apps/frontend/src/app.scss

```diff
@@ -179,33 +190,47 @@ nav {
     }
 
     & > table {
+      display: flex;
```

Comment:
All the scaling stuff is kinda useless cause I already fixed it in #171 + I don't really like the way the bottom control panel scrolls independently from the rest of the screen (you can basically scroll below the control panel when >100% zooming; see image below)
![image](https://user-images.githubusercontent.com/54808295/213868554-a1b24cb1-ae1a-4b48-907e-36c8ce1a99a6.png)

Reply:
I'll make sure to look into your styling changes when merging your and #171's branch, but I think especially the responsiveness is not handled perfectly here.

Reply:
We will use yours as discussed